blocks/v2-video/v2-video.css

```diff
@@ -84,6 +84,37 @@ main .section.v2-video-container .v2-video-wrapper {
   line-height: var(--f-heading-5-line-height);
 }
 
```

Comment:
I think this should more generic rather than specific to v2-video.css. Because if there is a video component embedded in other block and page doesn't have v2-video block the styles don't show up.
Ex: content cards can have media. I see there is html present , however styles are missing
https://182-video-pause-button--vg-volvotrucks-us-rd--netcentric.hlx.page/drafts/syb/teaser-cards

Reply:
Video playback controls are now added to key facts and teaser cards.
Styles has been removed from v2-video.